Makefile

```diff
@@ -0,0 +1,11 @@
+build:
+	docker-compose build --no-rm --parallel
+
+db:
+	docker-compose up -d --remove-orphans postgres
+
+test: db
```

Comment:
We could specify the dependency in the `docker-compose` file, like we do in aleph: https://github.com/alephdata/aleph/blob/64800a9068e46b2d33abc37baa0cdb7dd8b263f0/docker-compose.dev.yml#L89